geth-poa/docker-compose.yml

```diff
@@ -124,6 +124,44 @@ services:
           }
         ]
 
+  sl-node3:
```

Comment:
Might be worth naming this `sl-member` or similar to make it clear it's a different type of node than signers

Similarly we could rename `sl-node1` to `sl-signer1` etc. but not super important